resources/tk_nuke_quickreview.nk

```diff
@@ -8,11 +8,11 @@ Group {
  xpos -338
  ypos -49
  mapsize {0.15 0.15}
- addUserKnob {20 tanktab l "ShotGrid Review"}
+ addUserKnob {20 tanktab l "Flow Production Tracking Review"}
```

Comment:
Missing change on line 5 for this file

![image](https://github.com/shotgunsoftware/tk-nuke-quickreview/assets/123113322/2f0ab44c-6582-4ab5-857e-79b895143bb3)